app.js

```diff
@@ -64,20 +64,42 @@ var appName = "thimble",
     ], {
       autoescape: true
     }),
-    parameters = require('./lib/parameters')(env),
-    routes = require('./routes')( utils, env, nunjucksEnv, appName ),
-    webmakerAuth = new WebmakerAuth({
-      forceSSL: env.get('FORCE_SSL'),
-      loginHost: env.get('APP_HOSTNAME'),
-      loginURL: env.get('LOGIN_URL'),
-      authLoginURL: env.get('LOGIN_URL_WITH_AUTH'),
-      secretKey: env.get('SESSION_SECRET'),
-      domain: env.get('COOKIE_DOMAIN')
-    }),
-    webmakerProxy = require('./lib/proxy')(env),
+    parameters = require('./lib/parameters')(),
+    routes = require('./routes')( utils, nunjucksEnv, appName ),
+    webmakerProxy = require('./lib/proxy')(),
 
     bracketsPath;
 
+function configuredCookieSession() {
+  var options = {
+    key: "mozillaThimble",
+    secret: env.get('SESSION_SECRET'),
+    cookie: {
+      expires: false,
```

Comment:
I tore this logic out of webmaker's shared auth module, which means this is how the entire webmaker suite is working at the moment. It might be worth thinking through though...